packages/launch/src/index.tsx

```diff
@@ -2,3 +2,4 @@
  * Internal dependencies
  */
 export { default } from './launch';
+export * as hooks from './hooks';
```

Comment:
Though it feels safer to encapsulate all hooks in `hooks`, it wouldn't be such a bad idea to just export all the hooks directly as first-level exports. That way there is no need to do `launchHooks.useXXX()` in all other places.

Reply:
Yes, they are also easily identifiable because of the naming (`use...`).

Reply:
Hooks have been changed to a first-level export (no `hooks` namespace)